apps/L1G/tinder/tinder-frontend/specs/ChatPerson.spec.tsx

```
@@ -0,0 +1,71 @@
+
+jest.mock('next/image', () => ({
+  __esModule: true,
+  default: (props) => {
+    return <img {...props} />;
+  },
+}));
+
+jest.mock('@/components/ChatWindow', () => {
+  return function MockChatWindow() {
+    return <div data-testid="chat-window">ChatWindow Component</div>;
+  };
```

ESLint

[error] Function component is not an arrow function

(react/function-component-definition)